src/PortfolioGenerator.java

```diff
@@ -1,5 +1,48 @@
+import java.util.Scanner;
+
 public class PortfolioGenerator {
     public static void main(String[] args) {
-        // Write your code here ... ⬇
+        
+	String name;
+	String emailAddress;
+	int progLanguages;
+	String personalInfo;
+	int achievementShare;
+	
+	Scanner in = new Scanner(System.in);
+
+	System.out.print("Please enter your name: ");
+	name = in.nextLine():
```

Comment:
I don't see you've used the safeInput library any where here. 
you need to use that to get the user input.

Reply:
Also that wasn't a semi-colon
```suggestion
	name = in.nextLine();
```